atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -344,6 +344,40 @@ void WebFrame::SetIsolatedWorldHumanReadableName(int world_id,
       world_id, blink::WebString::FromUTF8(name));
 }
 
+void WebFrame::SetIsolatedWorldInfo(int world_id, mate::Arguments* args) {
+  if (args->Length() < 1) {
+    args->ThrowError("Invalid args");
+    return;
+  }
+
+  v8::Local<v8::Value> val;
+  args->GetNext(&val);
```

Comment:
No need to extract to a local handle explicitly here, attempting something like:

```
mate::Dictionary options;
if (!args->GetNext(&options))
```

should handle the conversion.